lib/android/src/main/java/com/wix/reactnativeuilib/dynamicfont/DynamicFontPackage.java

```diff
@@ -0,0 +1,27 @@
+package com.wix.reactnativeuilib.dynamicfont;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import com.facebook.react.ReactPackage;
+import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.uimanager.ViewManager;
+import com.facebook.react.bridge.JavaScriptModule;
+public class DynamicFontPackage implements ReactPackage {
```

Comment:
missing new line here and in the edn of the file

Reply:
Fixed